backend/tests/students.test.js

```diff
@@ -0,0 +1,77 @@
+import request from 'supertest';
+import app from '../app';
+import db, { sequelize } from '../models';
+// import Student from '../models/Student.model.js';
+
+const { Student } = db;
+
+describe('GET /api/students', () => {
+  beforeAll(async () => {
+    try {
+      console.log('Syncing database...');
+      await sequelize.sync({ force: true }); // Sync the database and clear existing data
+      console.log('Database synced successfully!');
```

Comment:
Since we're probably going to need `sequelize.sync`  in other tests as well, can you move this to a global setup in jest? That way it'll only be done once across all tests

See this in the jest documentation on how to do that: https://jestjs.io/docs/configuration#globalsetup-string